packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -1,14 +1,24 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-// @ts-expect-error need to migrate helpers to TypeScript
-import { createClientRender, fireEvent, screen, ErrorBoundary, createEvent } from 'test/utils';
+import {
+  createClientRenderStrictMode,
+  // @ts-expect-error need to migrate helpers to TypeScript
+  fireEvent,
+  // @ts-expect-error need to migrate helpers to TypeScript
+  screen,
+  // @ts-expect-error need to migrate helpers to TypeScript
+  ErrorBoundary,
+  // @ts-expect-error need to migrate helpers to TypeScript
+  createEvent,
+} from 'test/utils';
```

Comment:
How long would it take to migrate them?

Reply:
I don't know, I have never looked at it. I think that we should ask ourselves if it's the best use of our time. So far, I think that the main value proposition of writing the tests in TypeScript is to increase the types coverage of the public API of the components. To make sure the types are solid.